api/src/main/java/org/pmiops/workbench/notebooks/NotebooksServiceImpl.java

```diff
@@ -281,10 +281,7 @@ public KernelTypeEnum getNotebookKernel(
 
   private Blob getBlobWithSizeConstraint(String bucketName, String notebookName) {
     Blob blob =
-        cloudStorageClient.getBlob(
-            bucketName,
-            NotebookUtils.withNotebookPath(
-                NotebookUtils.withJupyterNotebookExtension(notebookName)));
+        cloudStorageClient.getBlob(bucketName, NotebookUtils.withNotebookPath(notebookName));
```

Comment:
I think what you are asking is can we test so that we are certain that we are getting blobs with the extensions that we expect.

In NotebooksServiceTest.java, I added expect statements to eight different preexisting tests, in order to ensure that the filename that we are asking for is the one that we want.

Please let me know if this covers what you are looking for.